src/index.ts

```diff
@@ -0,0 +1,11 @@
+class Challenge 
+{
+    buzz(input: number): Promise<string | number> 
+    {
+        return new Promise(resolve => 
+            resolve(`${input % 3 === 0 ? "fizz" : ""}${input % 5 === 0 ? "buzz" : ""}` || input)
```

Comment:
:mage_man:

Reply:
You could have used `Promise.resolve(/* stuff */)` too :sunglasses:

Reply:
Was the promise requested by the test of the exercise?

Reply:
Yes, it was